src/components/modals/NewMemberModalHeader.jsx

```diff
@@ -5,10 +5,21 @@ import { ReactComponent as Twitter } from "@unioncredit/ui/lib/icons/twitter-fil
 import JSConfetti from "js-confetti";
 import { useEffect, useRef } from "react";
 import { Text, Heading, ButtonRow, Button } from "@unioncredit/ui";
+import getProfileUrl, { generateTwitterLink } from "utils/generateLinks";
+import { useAccount, useNetwork } from "wagmi";
+import { useMember } from "providers/MemberData";
+import { ZERO } from "constants";
+import format from "utils/format";
 
 export default function NewMemberModalHeader() {
+  const { chain } = useNetwork();
+  const { address } = useAccount();
+  const profileUrl = getProfileUrl(address, chain.id);
```

Comment:
Nit: maybe we can move this to line 24 I think it's nice to try and group hooks together